src/server/handlers/github-auth.js

```diff
@@ -3,7 +3,7 @@ import logging from '../logging';
 
 import { conf } from '../helpers/config';
 
-const AUTH_SCOPE = 'admin:repo_hook';
+const AUTH_SCOPE = 'admin:repo_hook'; // another scope (like 'public_repo') may be needed to list organisations' repositories
```

Comment:
It seems that current OAuth scope we use allows only access to repositories owned by the user (not by organisations). If we want to list all repositories user has access to we need to add another scope to authorisation (probably `public_repo` should be enough [[docs](https://developer.github.com/v3/oauth/#scopes)]).

Another issue is that it doesn't limit repositories to those that user is admin of - and we require admin privileges to create hooks. So we may get some repositories (from organisations or collaboration) that user has access to, but is not admin of and can't create hooks.

I couldn't find any way through API to list only the repositories that user is admin of, so we will probably need to filter it separately or simply check at the time when user wants to build given repo (to avoid API calls for each repo from the list separately).